automation/check-patch.e2e-kubevirt-ipam-controller-functests.sh

```diff
@@ -53,6 +53,8 @@ main() {
     deploy_cnao
     deploy_cnao_cr
     ./hack/deploy-kubevirt.sh
+    ./cluster/kubectl.sh -n kubevirt patch kubevirt kubevirt --type=json --patch '[{"op":"add","path":"/spec/configuration/developerConfiguration","value":{"featureGates":[]}},{"op":"add","path":"/spec/configuration/developerConfiguration/featureGates/-","value":"NetworkBindingPlugins"}]'
```

Comment:
please consider moving this to a function like `PatchManagedMacvtapNetworkBinding()`

Reply:
i dont think we need, it is just 2 lines and the whole file doesnt have funcs atm

Reply:
It is not needed. it would just make it more readable. 
your call.